install_config/persistent_storage/persistent_storage_gce.adoc

```diff
@@ -142,3 +142,35 @@ formatted with the given file system.
 
 This allows using unformatted GCE volumes as persistent volumes, because
 {product-title} formats them before the first use.
+
+=== Multizone Configuration
+
+In multi-zone configurations, You must specify
```

Comment:
You must specify if you are not running one kubernetes cluster per GCE ProjectID. It is generally preferred to run a single cluster and only a single cluster inside a GCE ProjectID so specification is not required.